lookout/core/tests/test_metrics.py

```diff
@@ -0,0 +1,192 @@
+import threading
+import unittest
+
+import requests
+
+from lookout.core.metrics import ConfidentCounter, PreciseFloat, start_prometheus, submit_event
+
+
+class MetricReader:
+    def __init__(self, port, addr="localhost"):
+        self.__is_running = False
+        self.__port = port
+        self.__addr = addr
+        self.__metrics = {}
+        self.__data = {}
+
+    @property
+    def port(self) -> int:
+        return self.__port
+
+    @property
+    def addr(self) -> str:
+        return self.__addr
+
+    @property
+    def metrics(self) -> dict:
+        return self.__metrics
+
+    @property
+    def data(self) -> dict:
+        return self.__data
+
+    def query_data(self, addr: str = None, port: int = None) -> str:
+
+        addr = self.addr if addr is None else addr
+        port = self.port if port is None else port
+        api_endpoint = "http://{}:{}".format(addr, port)
+        r = requests.get(url=api_endpoint)
+        if r.status_code == 200:
+            data = r.content.decode()
+            return data
+        else:
```

Comment:
```suggestion
```
If you return, there is no "else"

Reply:
This really comes in handy when the server fails and does not return a 200 status. The way the server formats the error messages by default sucks.